packages/runtime/src/types.ts

```diff
@@ -23,6 +23,8 @@ export type TEALStack = IStack<bigint | Uint8Array>;
 
 export interface EncTx extends EncodedTransaction {
   txID: string
+  approvalProgram?: string
+  clearProgram?: string
```

Comment:
Please add a note that we expect TEAL code as a full string here (not the compiled version). Which should we taken as input from the user itself